docs/source/user_guide/getting_started_with_dpf_server.rst

```diff
@@ -0,0 +1,136 @@
+	
+PyDPF-Core and PyDPF-Post python modules can now be used. The instructions to install and get started with PyDPF-Core 
+(ansys-dpf-core module) can be found at `Getting Started section <https://dpf.docs.pyansys.com/getting_started/install.html>`_. 
+
+Using DPF Server
+----------------
+
+Running the DPF Server
+~~~~~~~~~~~~~~~~~~~~~~
+
+On Windows, start the DPF Server by running the Ans.Dpf.Grpc.bat file in the unzipped package.
+On Linux, start the DPF Server by running the Ans.Dpf.Grpc.sh file in the unzipped package.
+
+Running the DPF Server in a Docker container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Along with the ansys_dpf_server_lin_v2023.2.pre0.zip archive mentioned in :ref:`Installing DPF Server <target_installing_server>`, download the Dockerfile.
+2. Copy both the archive and Dockerfile in a folder and navigate into that folder.
+3. To build the DPF Docker container, run the following commands:
+
+.. code::
+
+    docker build . -t dpf-core:v2023_2_pre0 --build-arg DPF_VERSION=232 --build-arg DPF_SERVER_FILE=ansys_dpf_server_lin_v2023.2.pre0.zip
+
+4. To run the DPF Docker container, see the :ref:`License terms<target_to_license_terms>` section.
+
+License terms
+-------------
+
+DPF Preview License Agreement 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _target_to_license_terms:
+
+DPF Server is protected using license terms specified in the DPFPreviewLicenseAgreement file that 
+can be found on the Ansys Customer Portal.
+
+To accept the DPF User Licensing Agreement terms, the following environment flag must be set: 
+
+.. code::
+
+    "ANSYS_DPF_ACCEPT_LA=Y"
+
+ANSYS_DPF_ACCEPT_LA confirms your acceptance of the DPF User Licensing Agreement. By passing the value "Y" to the environment variable 
+"ANSYS_DPF_ACCEPT_LA", you are expressing that you have a valid and existing license for the edition and version of DPF server you intend to use.
+
+For a DPF Docker container usage, it can be set using:
+
+.. code::
+
+    docker run -e "ANSYS_DPF_ACCEPT_LA=Y" -e ANSYSLMD_LICENSE_FILE=1055@<license_server_to_use> -p 50052:50052 -e DOCKER_SERVER_PORT=50052 --expose=50052 dpf-core:v2023_2_pre0
+
+For any other case, set "ANSYS_DPF_ACCEPT_LA" as an environment variable with "Y" value.
+
+Replace "<license_server_to_use>" mention that ANSYSLMD_LICENSE_FILE environment variable points to the Ansys license server.
+For more information about Ansys license mechanism use with DPF Server, see :ref:`Ansys licensing section<target_to_ansys_license_mechanism>`.
+
+
+Ansys licensing
+~~~~~~~~~~~~~~~
+
+.. _target_to_ansys_license_mechanism:
+
+DPF Server is protected by Ansys licensing mechanism.
+
+DPF capabilities are available through the following main services: 
+- Entry: Loads the minimum number of plugins for basic use. It is the default. It will only check if an Ansys License is available. 
+- Premium: Uses the premium DataProcessingCore.xml file to load most plugins.... with their environments. It will checkout a license. 
```

Comment:
```suggestion
- Premium: Uses the premium DataProcessingCore.xml file to load most plugins with their environments. It will checkout a license. 
```